arm_robots/launch/med.launch

```diff
@@ -0,0 +1,66 @@
+<launch>
+<!--    <env name="ROSCONSOLE_CONFIG_FILE"-->
+<!--         value="$(find arm_robots)/config/victor_rosconsole.conf"/>-->
```

Comment:
better to make a version of this than comment it out. you can also delete it. But don't leave it commented out